<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addticketdialog</class>
 <widget class="QDialog" name="addticketdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>301</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>CONTACT DETAILS</string>
   </property>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="lb_name_2">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_email_2">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_phone_2">
       <property name="text">
        <string>Phone</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>191</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLineEdit" name="lineEdit_13"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_14"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_15"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>401</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string> TICKET DETAILS</string>
   </property>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>105</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QLabel" name="lb_ticket_num_2">
       <property name="text">
        <string>Ticket Number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_incident_cat_2">
       <property name="text">
        <string>Incident Category</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_tags_2">
       <property name="text">
        <string>Tags</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_impact_2">
       <property name="text">
        <string>Impact</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_urgency_2">
       <property name="text">
        <string>Urgency</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_priority_2">
       <property name="text">
        <string>Prioity</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_timestamp_2">
       <property name="text">
        <string>Timestamp</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_level_support_2">
       <property name="text">
        <string>Level of Support</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>241</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_7"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_8"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_9"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_10"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_11"/>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_12"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_symptoms_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>103</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>SYMPTOMS</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtSymptoms">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>311</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Uni Sans Heavy CAPS</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(2, 173, 237);</string>
   </property>
   <property name="text">
    <string>TICKET SUBMISSION</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtRating">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>240</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Rating</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
